README.md

```diff
@@ -15,13 +15,24 @@ The (internal) project-board is [here](https://github.com/orgs/hpi-sam/projects/
 5. Run `npm run setup` from the root folder
 6. (Optional) We have a list of recommended [vscode](https://code.visualstudio.com/) extensions. We strongly recommend you to use them if you are developing. You can see them via [the `@recommended` filter in the extensions panel](https://code.visualstudio.com/docs/editor/extension-marketplace#_recommended-extensions).
 
```

Comment:
If you follow all the installation steps and then the steps under "Starting for development" you should end up with a working application.
This is not the case anymore.

Add something like this to installation:
6. Install PostgresSQL (version 14) from here.
7. Setup PostgresSQL (To confirm with: "It must use a configuration according to the environment variables specified in `backend/.env` for name, user, password, host and port.")
8. Run the migrations (+ link to backend README).

You can also link to other sections of the documentation for more info/shortcuts  "Instead of 6. and 7. you can also start the backend without a database (See here) or let it run in docker (See here)").